src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -254,29 +351,44 @@ static void CheckForExperimentStatus(ITextView textView, in CompletionOptions op
             {
                 AsyncCompletionLogger.LogExpanderUsage();
 
+                // It's possible we didn't provide expanded items at the beginning of completion session because it was slow even if the feature is enabled.
+                // ExpandedItemsTask would be available in this case, so we just need to return its result.
+                // We are doing this in a spinning fashion because the lifetime of expandedItemsTask is associated with the completion session, we don't
```

Comment:
i don' t know what 'spinning fashion' means.

Reply:
The code has changed but I forgot to update the comment to reflect that